### PR TITLE
[wdspec] doc adding new WebDriver BiDi modules

### PR DESCRIPTION
**Pull request by sadym-chromium**
Explain adding WebDriver BiDi modules in example of [Permissions](https://www.w3.org/TR/permissions/).

### CODE REVIEW

File: docs/writing-tests/wdspec.md [hunk 1]
```diff
@@ -66,3 +67,97 @@ a WebDriver session) are defined in terms of Pytest "fixtures" and must be
...
+
+This section describes how to extend the WebDriver BiDi client with an example of
+adding support for [Web Bluetooth](https://webbluetoothcg.github.io/web-bluetooth).
+
+#### Adding a New Module
+
+##### Create `BidiModule`
+
+BiDi modules are defined in the `tools/webdriver/webdriver/bidi/modules/` directory.
+To add a new module called `bluetooth`, declare a Python class
```
Comment (jgraham) [inline docs/writing-tests/wdspec.md:94]: I would quite strongly prefer the examples here to be something that has multi-vendor support.

**Review by jgraham — CHANGES_REQUESTED** [review]

File: docs/writing-tests/wdspec.md (hunk 1 above)
Comment (whimboo) [reply docs/writing-tests/wdspec.md:94]: I would agree here. The `Permissions` API can be used instead for such an example. It's one which has cross-browser support.

File: docs/writing-tests/wdspec.md [hunk 2]
```diff
@@ -66,3 +67,97 @@ a WebDriver session) are defined in terms of Pytest "fixtures" and must be
...
+`webdriver.bidi.modules.bluetooth.Bluetooth` that inherits from `BidiModule` and
+store it in `tools/webdriver/webdriver/bidi/modules/bluetooth.py`:
```
Comment (whimboo) [inline docs/writing-tests/wdspec.md:96]: Should we store modules for external specifications under `external/`? We probably won't have naming glitches but that would clearly prevent it in the future.

File: docs/writing-tests/wdspec.md (hunk 1 above)
Comment (OrKoN) [reply docs/writing-tests/wdspec.md:94]: I am not sure this guide requires a specific module as an example, I would be happy with a `foo` module using a `bar` command but permissions sounds good to me too.

Comment (sadym-chromium) [reply docs/writing-tests/wdspec.md]: Switched to `Permissions`, as it allows to provide a specific code snippets.

**Review by jcscottiii — APPROVED** [review]
Thanks for this. LGTM.

**Review by OrKoN — APPROVED** [review]

File: docs/writing-tests/wdspec.md (hunk 1 above)
Comment (sadym-chromium) [reply docs/writing-tests/wdspec.md]: @jgraham was it the only blocker for the spec?

**Review by Mahmoudhpp — APPROVED** [review]

**Review by jgraham — APPROVED** [review]
Approved subject to the suggestion or similar being adopted.

**Pull request merged**